src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -277,17 +275,21 @@ public async Task InitializeAsync(CancellationToken cancellationToken = default)
                 PreInitialize();
                 HostInitializing?.Invoke(this, EventArgs.Empty);
 
-                _workerRuntime = _workerRuntime ?? _environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionWorkerRuntime);
```

Comment:
Note -- having `_workerRuntime` as a field was confusing as it was updated in multiple places. I've made it into a local here so this is the only flow where this can be determined.